templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -331,7 +331,19 @@ contents:
 
     stop_all_containers() {
       echo "Stopping all containers.."
-      crictl ps -q | xargs -r crictl stop
+      conids=$(crictl ps -q)
+      iterations=0
+      while [ ! -z "$conids" ]; do
+          let iterations=$iterations+1
+          if [ $iterations -ge 60 ]; then
+              echo "Failed to stop all containers after 60 iterations. Exiting!"
+              exit 1
+          fi
+          crictl stop $conids || true
+          echo "Waiting for all containers to stop... ($iterations/60)"
```

Comment:
Optional nit/question would this be more informative to folks at the start of the loop?

Reply:
Could then also drop https://github.com/openshift/machine-config-operator/pull/1262/files#diff-df2f7b2c367c15c3aa83f8b8e002b3d8R333

Reply:
OK. I will get rid of lineno. 333.

Reply:
I removed 333, and added 346 (just so that if we don't even get into the while loop, we will have a message).